vaporetto/src/tag_trainer.rs

```diff
@@ -47,6 +47,7 @@ pub struct TagTrainer<'a> {
     _type_window_size: u8,
     type_ngram_size: u8,
     // Uses BTreeMap to improve compression ratio.
+    default_tags: HashMap<&'a str, &'a [Option<Cow<'a, str>>]>,
```

Comment:
```suggestion
    default_tags: HashMap<&'a str, &'a [Option<Cow<'a, str>>]>,
    // Uses BTreeMap to improve compression ratio.
```
I guess this comment should be one line down.